flowbased-computation/flowbased-computation-api/pom.xml

```diff
@@ -59,6 +63,12 @@
         </dependency>
 
         <!-- Runtime dependencies -->
```

Comment:
Remove scope tag (compile is default) and move the dependance in compile dependencies area of the file